### PR TITLE
Issue #4017: Allow permanent but unused files to be referenced.

### PR DESCRIPTION
**Pull request by Graham-72**
Checking this simple fix to see if tests pass.

### CODE REVIEW

File: core/modules/file/file.module [hunk 1]
```diff
@@ -1487,15 +1487,8 @@ function file_managed_file_validate(&$element, &$form_state) {
...
-      if ($file->status == FILE_STATUS_PERMANENT) {
-        $references = file_usage_list($file);
-        if (empty($references)) {
-          form_error($element, t('The file used in the !name field may not be referenced.', array('!name' => $element['#title'])));
-        }
-      }
-    }
-    else {
+    $file = file_load($element['fid']['#value']);
+    if ($file = false) {
```
Comment (docwilmot) [inline core/modules/file/file.module]: This is just setting `File` to `FALSE`; did you mean `if ($file == false)`?

Comment (Graham-72) [reply core/modules/file/file.module]: Yes! Ooops!

**Comment by indigoxela** [comment]
Very likely this approach has negative side effects regarding unmanaged files. See my [issue comment](https://github.com/backdrop/backdrop-issues/issues/4017#issuecomment-564945239)

**Comment by klonos** [comment]
Just trying something here:

Fixes: https://github.com/backdrop/backdrop-issues/issues/4017

**Comment by klonos** [comment]
...meh, it needs to be added in the first comment of the PR I guess.

**Comment by herbdool** [comment]
Original issue is closed with alt pr merged. Closing this

**Pull request closed without merging**